.github/workflows/ci.yaml

```diff
@@ -29,10 +29,24 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       version: ${{ steps.version.outputs.version }}
+      is_pull_request: ${{ steps.is_pull_request.outputs.is_pull_request }}
```

Comment:
One quesiton: In [`image.yaml`](https://github.com/NVIDIA/nvidia-container-toolkit/blob/09200187e5bbe84b9662904337a24941dbe90607/.github/workflows/image.yaml#L40-L41) we already have
```
        ispr:
          - ${{ github.ref_name != 'main' && !startsWith( github.ref_name, 'release-' ) }}
```
Is there a reason that we don't just use that here again too?